src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -45,6 +45,7 @@ internal FakeManager(Type fakeObjectType, object proxy)
                                          new CallRuleMetadata { Rule = new ObjectMemberRule(this) },
                                          new CallRuleMetadata { Rule = new AutoFakePropertyRule(this) },
                                          new CallRuleMetadata { Rule = new PropertySetterRule(this) },
+                                         new CallRuleMetadata { Rule = new CancellationRule() },

```

Comment:
My thinking was that it was very unlikely for a property to have a `CancellationToken` as an index, and if it did, it probably wouldn't mean what it usually means for a method, so I gave the `CancellationRule` a lower priority than the property rules. As for `ObjectMemberRules`, the members of `Object` don't accept cancellation tokens as argument, so there's no point in looking for them.